src/order.ts

```diff
@@ -23,11 +23,10 @@ import { ERC20 } from "../generated/OrderBook/ERC20";
 function getOffer(
   taker: Address,
   index: BigInt,
-  orderAddress: Address,
-  closedAt: BigInt
+  orderAddress: Address
 ): Offer {
   const offerId =
-    `${taker.toHex()}-${index.toString()}-${orderAddress.toHex()}-${closedAt.toString()}`;
+    `${taker.toHex()}-${index.toString()}-${orderAddress.toHex()}`;
```

Comment:
With `closedAt` I was solving the problem of a user submitting an offer at an index, that offer completing, then in the future them repeating with the same index. This is how our UI would work rn. It is a problem because the first and second times would have the same entityID which is bad. 

Does your change have a solution to this I am missing?

Reply:
yeah i add something here to solve this
https://github.com/rwtp/subgraph/pull/12/commits/b13869fd6db3f6316e8199a99a84b83d4dd72ac5

Reply:
Ooo that’s good